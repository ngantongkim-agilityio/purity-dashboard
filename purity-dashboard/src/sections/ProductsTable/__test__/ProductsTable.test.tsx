import { render } from '@testing-library/react';

// Components
import { ProductsTable } from '..';

jest.mock('@/services/products.ts', () => ({
  fetchFilteredProducts: jest.fn()
}));

describe('ProductsTable component', () => {
  const mockProps = {
    query: 'test',
    currentPage: 1
  };

  it('Should render ProductsTable component with default props', () => {
    const { container } = render(<ProductsTable {...mockProps} />);

    expect(container).toMatchSnapshot();
  });
});
