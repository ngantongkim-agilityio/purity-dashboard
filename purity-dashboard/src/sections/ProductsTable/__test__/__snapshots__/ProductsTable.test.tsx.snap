// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductsTable component Should render ProductsTable component with default props 1`] = `<div />`;
